src/menu/menu.scss

```diff
@@ -4,6 +4,98 @@
 @import '../inputs/variables';
 @import '../icon/index';
 
+@mixin iui-dropdown {
+  @include iui-reset;
+  @include iui-font-family;
+  width: 100%;
+  box-shadow: $iui-elevation-2;
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+}
+
+@mixin iui-list-menu {
```

Comment:
If all these styles are added here, they must be removed from somewhere else, right?

Reply:
All of this styling was originally a part of the mixin iui-menu.  I split it up into 2 mixins so I could reuse the background & drop shadow portion without bringing along the list styling.  GitHub seems to be marking way more content as new than should be.  Here is a before & after comparison in VSC.
![Screen Shot 2021-04-28 at 4 26 16 PM](https://user-images.githubusercontent.com/849817/116468109-5055f900-a825-11eb-9393-301df58852fa.png)

Reply:
Ok, I don't understand why `iui-list-menu` is needed because it almost the same as `iui-menu`. Are you sure you don't forgot to delete one or the other?
![menu_mixin](https://user-images.githubusercontent.com/36186912/116523120-8e632380-a8de-11eb-8dc4-43c649942afb.gif)

Reply:
Yeah, its like a duplicate again...

Reply:
You are correct, I forgot to delete the duplicate. 😅. Fixed in my last push.

Reply:
There you go 😄